rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbIntegrationTest.java

```diff
@@ -0,0 +1,567 @@
+package co.rsk.peg.feeperkb;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.core.RskAddress;
+import co.rsk.peg.InMemoryStorage;
+import co.rsk.peg.feeperkb.constants.FeePerKbConstants;
+import co.rsk.peg.feeperkb.constants.FeePerKbMainNetConstants;
+import co.rsk.peg.storage.StorageAccessor;
+import org.ethereum.core.SignatureCache;
+import org.ethereum.core.Transaction;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(Lifecycle.PER_CLASS)
+class FeePerKbIntegrationTest {
+    private final FeePerKbConstants feePerKbConstants = FeePerKbMainNetConstants.getInstance();
+    private FeePerKbSupport feePerKbSupport;
+    private SignatureCache signatureCache;
+    private Coin currentFeePerKb;
+    private Coin differentFeePerKbVote;
+
+    @BeforeAll
+    void setUp() {
+        StorageAccessor inMemoryStorage = new InMemoryStorage();
+        FeePerKbStorageProvider feePerKbStorageProvider = new FeePerKbStorageProviderImpl(inMemoryStorage);
+        feePerKbSupport = new FeePerKbSupportImpl(feePerKbConstants, feePerKbStorageProvider);
+        signatureCache = mock(SignatureCache.class);
+        differentFeePerKbVote = Coin.valueOf(60_000L);
```

Comment:
This field is only set to a value here; it seems to be a constant to me. It could be moved to fed-level declaration. Wdyt?

Reply:
Done! Yes, it is a constant. Good catch! differentFeePerKbVote was changed from variable to constant.